kibbeh/src/modules/user/FollowingPage.tsx

```diff
@@ -14,7 +14,7 @@ interface UserPageProps {}
 export const FollowingPage: PageComponent<UserPageProps> = ({}) => {
   return (
     <WaitForWsAndAuth>
-      <HeaderController embed={{}} title="Following" />
+      <HeaderController embed={{}} title="Peoples" />
```

Comment:
?

Reply:
Being honest, the `following` pages and `followers` page are combined into one and only have a different function. 

You can check to the [next.dogehouse.tv](next.dogehouse.tv) and go to the following and followers section. It's all have title: `Following | DogeHouse`.

So I generalized it into `Peoples | DogeHouse`, because I can't find a solution except creating separate pages for each one, (but the function is to messy for me).

Reply:
ok then peoples isn't grammatically correct
```suggestion
      <HeaderController embed={{}} title="People" />
```

Reply:
lgtm